krystal-visualization/src/main/resources/static/js/graphRenderer.js

```
@@ -0,0 +1,907 @@
+      .attr("marker-end", d => d.edgeData.canFanout ? "url(#arrow-fanout)" : "url(#arrow)")
+      .attr("data-source", d => d.sourceId)
+      .attr("data-target", d => d.targetId)
+      .attr("data-edge-id", d => d.edgeData.name)
+      .each(function(d) {
+        if (d.edgeData.canFanout) {
+          d3.select(this).append("title").text("This dependency can fan out to multiple targets");
+        }
+      })
+      .on("click", (event, d) => {
+        event.stopPropagation();
+        if (d.edgeData && d.edgeData.documentation && d.edgeData.documentation.trim() !== "") {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)